### ISSUE TITLE
Device selector improvements

### ISSUE THREAD

**Issue by joseivanlopez** [openSUSE/agama#585]
A new `DeviceSelector` widget was added, see https://github.com/openSUSE/agama/pull/578. The new component is not accessible (e.g., keyboard navigation) yet, and it should provide some kind of filter to make easier to find a device when there are too many options.

**Comment by dgdavid** [comment]
Some notes taken during the #578 development available at https://gist.github.com/dgdavid/b91f18c4f20cf1e2c34b715201c43e0b

**Comment by ancorgs** [comment]
Is this still relevant? We completely changed the disk selector already, so I will close it. But feel free to reopen if the problems still stand.
